src/layouts/index.vue

```diff
@@ -26,6 +26,19 @@ export default Vue.extend({
       version: GIT_REVISION_SHORT,
     };
   },
+  computed: {
```

Comment:
IMO, there is no sense to wrap up getter into the computed property. In that case you trade up extra reactive listener for a something like readability which actually don't effect the code